app/views/countries/data_countries.erb

```diff
@@ -0,0 +1,163 @@
+<h1>Countries Data</h1>
+
+<div class="panel panel-default">
+  <div class="panel-heading">Code to import data from API into <code>Daru::DataFrame</code> (yes, that's all!):</div>
+  <div class="panel-body">
+    <%=
+      rouge(%{
```

Comment:
It would be better if we put these lines of code (in string) and use it in view (in all the places), isn't it ?

Reply:
any update on this ?